### DISCUSSION TITLE
genesis in the repo is not the one expected by this version of Lotus!

### DISCUSSION THREAD

**Discussion by colccool** [filecoin-project/lotus#6732] [General Q&A]
run- lotus daemon

version: lotus version 1.11.1-dev+calibnet+git.88ac2ebc0

ERROR: initializing node: starting node: could not build arguments for function "github.com/filecoin-project/lotus/node/modules/lp2p".StartListening.func1 (/Users/pluto/lotus/node/modules/lp2p/addrs.go:98): failed to build host.Host: could not build arguments for function "reflect".makeFuncStub (/opt/homebrew/Cellar/go/1.16.5/libexec/src/reflect/asm_arm64.s:12): failed to build lp2p.BaseIpfsRouting: could not build arguments for function "reflect".makeFuncStub (/opt/homebrew/Cellar/go/1.16.5/libexec/src/reflect/asm_arm64.s:12): failed to build dtypes.NetworkName: could not build arguments for function "reflect".makeFuncStub (/opt/homebrew/Cellar/go/1.16.5/libexec/src/reflect/asm_arm64.s:12): failed to build dtypes.AfterGenesisSet: received non-nil error from function "reflect".makeFuncStub (/opt/homebrew/Cellar/go/1.16.5/libexec/src/reflect/asm_arm64.s:12): genesis in the repo is not the one expected by this version of Lotus!

someone can help me ?

**Comment by de-authority** [comment]
Usually, this error is because you have not cleared the old version of the compilation environment. I suggest you download the master branch code again and recompile it. If lotus daemon can run normal, please let me know

**Comment by de-authority** [comment]
Please  remember  to delete the old lotus folder before  you download lotus.

**Reply by colccool** [reply]
I delete the old folder and recompile it ,but the bug still exist

**Reply by de-authority** [reply]
You can try reinstall golang.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
go with a stable version for calibnet -the master might not be up to date with what is running there

**Comment by de-authority** [comment]
I encounter this err today cause I run lotus daemon when i didn't delete old data  which is store at ~/.lotus 
hope this can help you!

**Comment by LexLuthr** [comment]
@colccool DO you still need help with this or were you able to fix this error? If yes, then can you tell us how you fixed it?
